script.js

```diff
@@ -1,61 +1,79 @@
 const books = JSON.parse(localStorage.getItem('form')) || [];
 
-function displayBooks(book, index) {
-  const content = `
-    <div>
-        <p>${book.title}</p>
-        <p>${book.author}</p>    
-        <button id ="delete${index}">Remove</button>
-        <hr>
-    </div>
-    `;
-  return content;
+class Books {
+  constructor(author, title, books, button) {
+    this.author = author;
+    this.title = title;
+    this.books = books;
+    this.button = button;
+  }
+
+  addBook() {
+    const { title } = this;
+    const { author } = this;
+
+    const newBook = {
+      title,
+      author,
+    };
+
+    this.books.push(newBook);
```

Comment:
- [ ] [OPTIONAL] I'd suggest you use the `concat()` method ([more info here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat)) instead of mutating the array directly as it's considered bad practice and could cause data inconsistency when you directly manipulate the array (although it might seem very mundane here as your project grows this will definitely come in handy)
- You could refactor it to something like this:
 ```
this.books = this.books.concat(newBook)
```